ServerAdministration/DockerSwarm/Proxy.md

```diff
@@ -0,0 +1,83 @@
+# Proxy
+
+For the Giraf project we only have two public IP's that both has an DNS A name to srv.giraf.cs.aau.dk. The firewall settings for those ips are that only port ```80``` and port ```443``` are allowed through and since the different parts of the system uses other ports such as port ```5000``` and ```3306``` we need an reverse proxy to pass the traffic intended for those port into our Docker Swarm. In the following the configuration of the proxy will be elaborated upon.
```

Comment:
Maybe change "IP's that both has an DNS A name to" to "IPs that both have a DNS name to".
Maybe change "those ips" to "those IPs".
Maybe change "we need an reverse" to "we need a reverse".
Maybe change "for those port" to "for those ports".
Maybe change "into our Docker Swarm" to "into the Docker Swarm".
Maybe change "In the following the configuration" into "In the following, the configuration".